eternalcore/src/main/java/com/eternalcode/core/chat/ChatManagerCommand.java

```diff
@@ -18,12 +18,13 @@
 @Permission("eternalcore.chat")
 public class ChatManagerCommand {
 
+    private static final int CLEAR_LINE_COUNT = 64;
```

Comment:
Czemu akurat `64`?

Reply:
bo wcześniej też było 64

Reply:
IMO nie powinno to tak wyglądać, ta liczba jest za mała aby można było to nazwać czyszczeniem czatu + powinno być to konfigurowalne. można to potem zaimplementować, ale może teraz ustaw to na większą liczbę.